### PR TITLE
Update climate_HassClimateGetTemperature.yaml

### PR DESCRIPTION
**Pull request by lockenkop**
add sentences containing "heiß" and "kalt"
Would adding sentences with "Artikel" be helpful?
"Wie warm ist es in (der | dem) <area>
edit: understood common.yaml now, so answer is: it's allready implemented

### CODE REVIEW

File: sentences/de/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -7,4 +7,6 @@ intents:
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
           - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie warm ist es <area>"
+          - "Wie heiß ist es <area>"
+          - "Wie kalt ist es <area>"
```
Comment (mib1185) [inline sentences/de/climate_HassClimateGetTemperature.yaml:12]: these 3 can be merged into one

```suggestion
          - "Wie (warm | heiß | kalt) ist es <area>"
```

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @lockenkop

please also add a test for this sentence

File: sentences/de/climate_HassClimateGetTemperature.yaml (hunk 1 above)
Comment (lockenkop) [reply sentences/de/climate_HassClimateGetTemperature.yaml]: that makes sense, thanks

File: sentences/de/climate_HassClimateGetTemperature.yaml [hunk 2]
```diff
@@ -7,4 +7,4 @@ intents:
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
           - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie (warm | heiß | kalt) ist es <area>"
```
Comment (mib1185) [inline sentences/de/climate_HassClimateGetTemperature.yaml:10]: if you just want to ask "Wie warm ist es?", than the `area` needs to be optional

```suggestion
          - "Wie (warm | heiß | kalt) ist es [<area>]"
```

Comment (Scorpoon) [reply sentences/de/climate_HassClimateGetTemperature.yaml]: Resolved with #445

**Pull request closed without merging**